frontend/src/components/results/PlayerResultsItem.tsx

```diff
@@ -82,15 +82,17 @@ type PlayerResultsCardProps = {
   isCurrentPlayer: boolean,
   gameStartTime: string,
   color: Color,
+  numProblems: number,
   onViewCode: () => void,
 };
 
 function PlayerResultsItem(props: PlayerResultsCardProps) {
   const {
-    player, place, isCurrentPlayer, color, gameStartTime, onViewCode,
+    player, place, isCurrentPlayer, color, gameStartTime, numProblems, onViewCode,
   } = props;
 
   const score = useGetScore(player);
```

Comment:
Since score and `numProblems` are used in conjunction to calculate one value, I feel like there'd be a simpler way of doing this such that `numProblems` doesn't have to be passed in to the `ResultsTable`, `Podium`, and `PlayerResultsItem`.

Reply:
I can look into this after merging/pulling from main in the next series of PRs